data/sqlite3/init-tables.sql

```diff
@@ -0,0 +1,23 @@
+CREATE TABLE directors (did integer primary key, name varchar(32));
+CREATE TABLE movies (mid integer primary key, name varchar(64), year integer);
+CREATE TABLE director_movie (
+    did integer, mid integer,
+    FOREIGN KEY(did) REFERENCES directors(did),
+    FOREIGN KEY(mid) REFERENCES movies(mid)
+);
+
+CREATE UNIQUE INDEX movie_dirx on director_movie(did,mid);
+
+INSERT INTO directors (name) VALUES ("Jim Jarmusch"), ( "Tim Burton");
+INSERT INTO directors VALUES(3,'Lana Wachowski');
```

Comment:
Maybe don't mix single and double quotes? :)

Reply:
OTOH this can count as testing different quoting styles ;-)

Reply:
Well played... but then could we also test a movie by `Alejandro González Iñárritu`? :D

Reply:
Added ;-)